intl/msg_hash_us.c

```diff
@@ -74,206 +77,94 @@ int msg_hash_get_help_us_enum(enum msg_hash_enums msg, char *s, size_t len)
-                     "Plain file.");
+            strlcpy(s, msg_hash_to_str(MENU_ENUM_LABEL_HELP_FILE_BROWSER_PLAIN_FILE), len);
             break;
         case MENU_ENUM_LABEL_FILE_BROWSER_MOVIE_OPEN:
-            snprintf(s, len,
-                     "Video. \n"
-                             " \n"
-                             "Select it to open this file with the \n"
-                             "video player.");
+            strlcpy(s, msg_hash_to_str(MENU_ENUM_LABEL_HELP_FILE_BROWSER_MOVIE_OPEN), len);
             break;
         case MENU_ENUM_LABEL_FILE_BROWSER_MUSIC_OPEN:
-            snprintf(s, len,
-                     "Music. \n"
-                             " \n"
-                             "Select it to open this file with the \n"
-                             "music player.");
+            strlcpy(s, msg_hash_to_str(MENU_ENUM_LABEL_HELP_FILE_BROWSER_MUSIC_OPEN), len);
             break;
         case MENU_ENUM_LABEL_FILE_BROWSER_IMAGE:
-            snprintf(s, len,
-                     "Image file.");
+            strlcpy(s, msg_hash_to_str(MENU_ENUM_LABEL_HELP_FILE_BROWSER_IMAGE), len);
             break;
         case MENU_ENUM_LABEL_FILE_BROWSER_IMAGE_OPEN_WITH_VIEWER:
-            snprintf(s, len,
-                     "Image. \n"
-                             " \n"
-                             "Select it to open this file with the \n"
-                             "image viewer.");
+            strlcpy(s, msg_hash_to_str(MENU_ENUM_LABEL_HELP_FILE_BROWSER_IMAGE_OPEN_WITH_VIEWER), len);
             break;
         case MENU_ENUM_LABEL_FILE_BROWSER_CORE_SELECT_FROM_COLLECTION:
-            snprintf(s, len,
-                     "Libretro core. \n"
-                             " \n"
-                             "Selecting this will associate this core \n"
-                             "to the game.");
+            strlcpy(s, msg_hash_to_str(MENU_ENUM_LABEL_HELP_FILE_BROWSER_CORE_SELECT_FROM_COLLECTION), len);
             break;
         case MENU_ENUM_LABEL_FILE_BROWSER_CORE:
-            snprintf(s, len,
-                     "Libretro core. \n"
-                             " \n"
-                             "Select this file to have RetroArch load this core.");
+            strlcpy(s, msg_hash_to_str(MENU_ENUM_LABEL_HELP_FILE_BROWSER_CORE), len);
             break;
         case MENU_ENUM_LABEL_FILE_BROWSER_DIRECTORY:
-            snprintf(s, len,
-                     "Directory. \n"
-                             " \n"
-                             "Select it to open this directory.");
-            break;
-        case MENU_ENUM_LABEL_CACHE_DIRECTORY:
-            snprintf(s, len,
-                     "Cache Directory. \n"
-                             " \n"
-                             "Content decompressed by RetroArch will be \n"
-                             "temporarily extracted to this directory.");
+            strlcpy(s, msg_hash_to_str(MENU_ENUM_LABEL_HELP_FILE_BROWSER_DIRECTORY), len);
```

Comment:
Uh, you replaced a Cache directory help message with a plain file browser directory one?
Why?

Reply:
Ah, the Cache directory help message was deleted outright not replaced.
Sorry, my mistake.

Reply:
Yes, that entry already had a sublabel which contains same info, so I removed it from here. Sublabel text:
MENU_ENUM_SUBLABEL_CACHE_DIRECTORY,
"Archived content will be temporarily extracted to this directory."